.github/workflows/ci.yml

```diff
@@ -10,15 +10,15 @@ jobs:
   test-ubuntu:
     runs-on: ubuntu-latest
     timeout-minutes: 15
-    continue-on-error: ${{ startsWith(matrix.kotlin-version, '1.4.3') }}
+    continue-on-error: ${{ startsWith(matrix.kotlin-version, '1.5') }}
```

Comment:
1.5 should be the next version. We're not using any preview builds for now, so it doesn't really matter in case it'll be 1.4.40.